src/WorkOS.net/Services/DirectorySync/Entities/User.cs

```diff
@@ -86,6 +87,14 @@ public class User
         [JsonProperty("custom_attributes")]
         public Dictionary<string, object> CustomAttributes { get; set; }
 
+        /// <summary>
+        /// Grab User's primary email.
+        /// </summary>
+        public Email GetPrimaryEmail
```

Comment:
Can we remove the `Get` prefix since this is a getter property?

```suggestion
        public Email PrimaryEmail
```